libyara/modules/pe/pe.c

```diff
@@ -3010,6 +3459,24 @@ begin_declarations
     declare_integer("ordinal");
   end_struct_array("export_details")
 
+  begin_struct_array("import_details");
+    declare_string("library_name");
+    declare_integer("number_of_functions");
+    begin_struct_array("functions");
+      declare_string("name");
+      declare_integer("ordinal");
+    end_struct_array("functions");
+  end_struct_array("import_details");
+
+  begin_struct_array("delay_import_details");
```

Comment:
`delayed_import_details` is declared twice.

Reply:
Redundancy removed.